lighthouse-core/test/config/config-test.js

```diff
@@ -246,20 +246,20 @@ describe('Config', () => {
           }
         },
       ],
-    }), /no failureDescription and should/);
+    }), /no failureTitle and should/);
 
     assert.throws(_ => new Config({
-      audits: [basePath + '/missing-help-text'],
-    }), /meta.helpText property/);
+      audits: [basePath + '/missing-description'],
+    }), /meta.description property/);
 
     assert.throws(_ => new Config({
       audits: [
         class EmptyStringHelpTextAudit extends Audit {
           static get meta() {
             return {
-              name: 'empty-string-help-text',
-              description: 'description',
-              helpText: '',
+              id: 'empty-string-help-text',
```

Comment:
helptext -> description

and you fixed the one in valid-custom-audit already. :)

Reply:
oh gotcha :) fixed!